### PR TITLE
feat: add cycle to courses schema

### PR DESCRIPTION
**Pull request by ruilopesm**

### CODE REVIEW

File: data/courses.txt
Comment (danielsp45) [inline data/courses.txt:1]: Is it supposed for this file to contain all of the courses?

Comment (ruilopesm) [reply data/courses.txt:1]: I guess so. Which one(s) is(are) missing?

Comment (danielsp45) [reply data/courses.txt:1]: From the most known ones, Medicina is missing.
And there must be missing a bunch of smaller ones, unless our scope is only UMinho degrees.

Comment (ruilopesm) [reply data/courses.txt:1]: Our scope, for now, is only UMinho degrees.

Comment (danielsp45) [reply data/courses.txt:1]: Then just add medicina and it must be all good

**Review by danielsp45 — DISMISSED** [review]

File: data/courses.txt [hunk 1]
```diff
@@ -1,241 +1,58 @@
...
-Programa doutoral em Ciências da Comunicação
-Programa doutoral em Design de Moda
-Programa doutoral em Economia
-Programa doutoral em Estudos Culturais
-Programa doutoral em Gestão
-Programa doutoral em História
-Programa doutoral em Marketing e Estratégia
-Programa doutoral em Modernidades Comparadas: Literaturas, Artes e Culturas
-Programa doutoral em Sociologia
+Ciências da Educação
```
Comment (danielsp45) [inline data/courses.txt]: Move this lower to the courses begginning with `C`

**Review by danielsp45 — APPROVED** [review]

**Pull request merged**